lib/modules/datasource/docker/index.ts

```diff
@@ -647,6 +646,7 @@ export class DockerDatasource extends Datasource {
   private async getDockerApiTags(
     registryHost: string,
     dockerRepository: string,
+    currentValue?: string,
```

Comment:
I don't like how a generic currentValue param has been added here but the only way it's used is if the host is quay.io? If you imagine a future developer looking at this code and wondering if they need to provide currentValue then they'd need to trace through the code, do a git blame, find this PR, etc. I'd prefer this is done more clearly

Reply:
Yes, and add comments explaining the reasoning behind it's use.